src/haddock/modules/analysis/seletopclusts/__init__.py

```diff
@@ -14,13 +15,8 @@ class HaddockModule(BaseHaddockModule):
 
     name = RECIPE_PATH.name
 
-    def __init__(
```

Comment:
Black lint is killing us. Much better before, also contributions signatures.

Reply:
Yeah, because is not a matter of lint, is a matter of style :-)

Reply:
Mine or the lack of yours? 🔥